icssc-projects/peterportal-client
# Converted to typescrpit

Converted server from JS to TS

## Description
Added typescript support for server.

## Steps to verify/test this change:
1. Run npm start on server
2. Check if server runs properly
